internal/tiltfile/tiltfile_docker_compose_test.go

```diff
@@ -421,6 +442,30 @@ k8s_yaml('bar.yaml')`
 	assert.Equal(t, 2, len(f.loadResult.Manifests))
 }
 
+func TestDockerComposeAndK8sNameConflict(t *testing.T) {
```

Comment:
can you also add a test for the case where the k8s_yaml comes before the docker_compose?